estuary_updater/handlers/base.py

```diff
@@ -107,14 +120,16 @@ def get_or_create_build(self, identifier, original_nvr=None, force_container_lab
 
         build_params = {
             'epoch': build_info['epoch'],
-            'extra': json.dumps(build_info['extra']),
             'id_': str(build_info['id']),
             'name': build_info['package_name'],
             'release': build_info['release'],
             'state': build_info['state'],
             'version': build_info['version']
         }
 
+        if build_info['extra']:
```

Comment:
This might throw an exception if `build_info['extra']` is `None`. Could you please change it to `if build_info.get('extra'):`